PICMI_Python/particles.py

```diff
@@ -19,6 +19,7 @@ class PICMI_Species(_ClassWithInit):
     Species
       - particle_type=None: A string specifying an elementary particle, atom, or other, as defined in the openPMD 2 species type extension, openPMD-standard/EXT_SpeciesType.md
       - name=None: Name of the species
+      - method: One of 'Boris', 'Vay', 'Higuera', 'Li' (string)
```

Comment:
openPMD cross-link: I think for `Higuera` we do consider `Higuera-Cary` as a name, but it's still open.
https://github.com/openPMD/openPMD-standard/issues/226

Further values in openPMD:
- `free-streaming` (constantly moving with initial momentum)
- `LLRK4` (reduced Laundau-Lifshitz pusher via RK4 and classical radiation reaction, [doi:10.1016/j.cpc.2016.04.002](https://dx.doi.org/10.1016/j.cpc.2016.04.002))
https://github.com/openPMD/openPMD-standard/blob/upcoming-2.0.0/EXT_ED-PIC.md#particle-records-macroparticles
- `other:freeText` for other things a code supports

Reply:
Good suggestions!  Please take a look at the new version and give feedbacks or edit.